packages.md

```diff
@@ -128,14 +138,17 @@
 | [`azure-mgmt-subscription`]( sdk/subscription/azure-mgmt-subscription/setup.py )| [Readme](sdk/subscription/azure-mgmt-subscription/README.rst) | [Changelog](sdk/subscription/azure-mgmt-subscription/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-mgmt-subscription ) |
 | [`azure-mgmt-trafficmanager`]( sdk/trafficmanager/azure-mgmt-trafficmanager/setup.py )| [Readme](sdk/trafficmanager/azure-mgmt-trafficmanager/README.rst) | [Changelog](sdk/trafficmanager/azure-mgmt-trafficmanager/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-mgmt-trafficmanager ) |
+| [`azure-mgmt-vmwarecloudsimple`]( sdk/compute/azure-mgmt-vmwarecloudsimple/setup.py )| [Readme](sdk/compute/azure-mgmt-vmwarecloudsimple/README.rst) | [Changelog](sdk/compute/azure-mgmt-vmwarecloudsimple/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-mgmt-vmwarecloudsimple ) |
 | [`azure-mgmt-web`]( sdk/appservice/azure-mgmt-web/setup.py )| [Readme](sdk/appservice/azure-mgmt-web/README.rst) | [Changelog](sdk/appservice/azure-mgmt-web/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-mgmt-web ) |
 | [`azure-nspkg`]( sdk/core/azure-nspkg/setup.py )| [Readme](sdk/core/azure-nspkg/README.rst) |  N/A  | [PyPI]( https://pypi.org/project/azure-nspkg ) |
 | [`azure-servicebus`]( sdk/servicebus/azure-servicebus/setup.py )| [Readme](sdk/servicebus/azure-servicebus/README.rst) | [Changelog](sdk/servicebus/azure-servicebus/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-servicebus ) |
 | [`azure-servicefabric`]( sdk/servicefabric/azure-servicefabric/setup.py )| [Readme](sdk/servicefabric/azure-servicefabric/README.rst) | [Changelog](sdk/servicefabric/azure-servicefabric/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-servicefabric ) |
 | [`azure-servicemanagement-legacy`]( sdk/core/azure-servicemanagement-legacy/setup.py )| [Readme](sdk/core/azure-servicemanagement-legacy/README.rst) | [Changelog](sdk/core/azure-servicemanagement-legacy/HISTORY.rst) | [PyPI]( https://pypi.org/project/azure-servicemanagement-legacy ) |
 | [`azure-storage-blob`]( sdk/storage/azure-storage-blob/setup.py )| [Readme](sdk/storage/azure-storage-blob/README.md) |  N/A  | [PyPI]( https://pypi.org/project/azure-storage-blob ) |
-| [`azure-storage-file-share`]( sdk/storage/azure-storage-file-share/setup.py )| [Readme](sdk/storage/azure-storage-file-share/README.md) |  N/A  | [PyPI]( https://pypi.org/project/azure-storage-file-share ) |
+| [`azure-storage-file-share`]( sdk/storage/azure-storage-file-share/setup.py )| [Readme](sdk/storage/azure-storage-file-share/README.md) |  N/A  |  N/A  |
 | [`azure-storage-queue`]( sdk/storage/azure-storage-queue/setup.py )| [Readme](sdk/storage/azure-storage-queue/README.md) |  N/A  | [PyPI]( https://pypi.org/project/azure-storage-queue ) |
+| [`pylint-guidelines-checker`]( scripts/pylint_custom_plugin/setup.py )| [Readme](scripts/pylint_custom_plugin/README.md) |  N/A  |  N/A  |
```

Comment:
Why do we need to include this package?

Reply:
We don't it was just in the repo :) I will clean that up.